common/static/common/js/utils/clamp-html.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Used to ellipsize a section of arbitrary HTML after a specified number of words.
+ *
+ * Note: this will modify the DOM structure of root in place.
+ * To keep the original around, you may want to save the result of cloneNode(true) before calling this method.
+ *
+ * Known bug: This method will ignore any special whitespace in the source and simply output single spaces.
+ * Which means that &nbsp; will not be respected. This is not considered worth solving at time of writing.
+ *
+ * Returns how many words remain (or a negative number if the content got clamped)
+ */
+function clampHtmlByWords(root, wordsLeft)
+{
```

Comment:
smallest of nits:I'm sure the linter doesn't care and I don't, but I feel like we should be consistent with our opening brackets either starting on the declaration of functions/conditions, or returning on the next line

Reply:
If I have to run quality again, I'll fix that